hw7/task1/builder.js

```diff
@@ -0,0 +1,15 @@
+class Builder {
+    constructor(string){
+        this.string = string;
+    }
+    append(data){
+        this.string += data;        
+    };    
+    build(){
+        throw new Error(`В ${this.constructor.string} не описан метод Builder()`);
```

Comment:
errors should have a description on English

Reply:
fixed